tests/tests/noir/proof_tests/shamir.rs

```diff
@@ -0,0 +1,74 @@
+use ark_bn254::Bn254;
+use co_ultrahonk::prelude::{
+    CoUltraHonk, HonkProof, ProvingKey, ShamirCoBuilder, SharedBuilderVariable,
+    UltraCircuitVariable, Utils,
+};
+use mpc_core::protocols::shamir::ShamirProtocol;
+use std::thread;
+use tests::shamir_network::ShamirTestNetwork;
+
+fn proof_test(name: &str, num_parties: usize, threshold: usize) {
+    const CRS_PATH_G1: &str = "../co-noir/ultrahonk/crs/bn254_g1.dat";
```

Comment:
can we always make these based on CARGO_MANIFEST_DIR such that this is not dependant on the location of where the test is run.
see https://github.com/TaceoLabs/collaborative-circom/blob/944550bf73022acb207e1e1c780e393446e6bdc3/co-circom/co-circom/benches/zkey_parse.rs#L6

Reply:
This is also a generell thing, not just for co-noir, so I would propose to do this at a later point for all